src/msg/async/Stack.cc

```diff
@@ -45,6 +47,7 @@ void NetworkStack::add_thread(unsigned i, std::function<void ()> &thread)
           // TODO do something?
         }
```

Comment:
I think we could call rcu_quiescent_state here when r == 0 means no active event happen